doc/devices/dts/device_tree.rst

```diff
@@ -187,6 +187,12 @@ The following is a more precise list of required files:
     underlying DTS derived configuration #defines.  Fixup files are temporary
     artifacts until additional DTS changes are made to make them unnecessary.
 
+* Overlay Files (optional)
+
+  * Overlay files contain tweaks or changes to the SoC and Board support files
+    described above. They can be used to modify Device Tree configurations
+    without having to change the SoC and Board files.
```

Comment:
Nitpick: a `:ref:` to the right section in application.rst would make this more useful.